distribution/ddf-common/src/main/resources/etc/org.apache.karaf.command.acl.shell.cfg

```diff
@@ -0,0 +1,15 @@
+#
+# This configuration file defines the ACLs for commands in the shell subshell
+#
+
+* = NO_ACCESS
+complete = admin
+echo = admin
+format = admin
+grep = admin
+if = admin
+keymap = admin
+less = admin
+set = admin
+setopt = admin
+while = admin
```

Comment:
I know some scripts we have currently also use `wc` and `tac` so we might want it include them here.